test/jdk/ProblemList.txt

```diff
@@ -623,6 +623,7 @@ com/sun/security/sasl/gsskerb/ConfSecurityLayer.java            8039280 generic-
 com/sun/security/sasl/gsskerb/NoSecurityLayer.java              8039280 generic-all
 sun/security/provider/PolicyFile/GrantAllPermToExtWhenNoPolicy.java 8039280 generic-all
 sun/security/provider/PolicyParser/PrincipalExpansionError.java 8039280 generic-all
+javax/net/ssl/SSLSession/CertMsgCheck.java                      8326699 generic-all
```

Comment:
You will need a separate bug id with failure to mention here. Can't use same bug id as problem listing.

Reply:
Filed https://bugs.openjdk.org/browse/JDK-8326705 for you to mention here.